tls/s2n_security_policies.c

```diff
@@ -494,6 +507,7 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version="20190121", .security_policy=&security_policy_20190121, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="20190122", .security_policy=&security_policy_20190122, .ecc_extension_required=0, .pq_kem_extension_required=0 },
     { .version="20190801", .security_policy=&security_policy_20190801, .ecc_extension_required=0, .pq_kem_extension_required=0 },
+    { .version="20190802", .security_policy=&security_policy_20190802, .ecc_extension_required=0, .pq_kem_extension_required=0 },
```

Comment:
Do you need to update the documentation for this? How do customers discover / understand this policy?

Reply:
Ref: https://github.com/awslabs/s2n/issues/1856